megamek/src/megamek/client/ui/swing/MegaMekGUI.java

```diff
@@ -749,19 +749,20 @@ public String getDescription() {
 
         // calculate initial BV
         server.calculatePlayerBVs();
-
+        
         // setup any bots
         for (int x = 0; x < pa.length; x++) {
             if (sd.playerTypes[x] == ScenarioDialog.T_BOT) {
```

Comment:
Let us swap this to a switch statement, with an empty default

Reply:
Let's make it an else if. Easy change and there are only three options. Testbot, princess, nothing.

Reply:
Don't worry about TestBot, we're going to get rid of it in the 0.49 line. (Or should, it doesn't work)

Reply:
New commit added to change to else if and switch Princess to be "Bot".  In prep for getting rid of TestBot (Then we can just drop Other Bot from the dropdown in ScenarioDialog and that is the extent of change).